ms2query/s2v_functions.py

```diff
@@ -0,0 +1,63 @@
+from matchms.filtering import normalize_intensities
+from matchms.filtering import require_minimum_number_of_peaks
+from matchms.filtering import select_by_mz
+from matchms.filtering import select_by_relative_intensity
+from matchms.filtering import reduce_to_number_of_peaks
+from matchms.filtering import add_losses
+
+
+def set_spec2vec_defaults(**settings):
+    """Set spec2vec default argument values"(where no user input is given)".
+
+    Args
+    ----------
+    **settings: optional dict
+        Change default settings
+    """
+    defaults = {"mz_from": 0,
+                "mz_to": 1000,
+                "n_required": 10,
+                "ratio_desired": 0.5,
+                "intensity_from": 0.001,
+                "loss_mz_from": 5.0,
+                "loss_mz_to": 200.0}
+
+    # Set default parameters or replace by **settings input
+    for key in defaults:
+        if key not in settings:
+            settings[key] = defaults[key]
+    return settings
+
+
+def post_process_s2v(spectrum, **settings):
+    """Returns a processed matchms.Spectrum.Spectrum
+
+    Args:
+    ----------
+    spectrum: matchms.Spectrum.Spectrum
+        Spectrum to process
+    **settings: optional dict
+        Change default settings
```

Comment:
```suggestion
    mz_from
        Set lower threshold for m/z peak positions. Default is 0.0.
    mz_to
        Set upper threshold for m/z peak positions. Default is 1000.0.
    n_required
        Number of minimal required peaks for a spectrum to be considered.
    ratio_desired
        Number of minimum required peaks. Spectra with fewer peaks will be set
        to 'None'. Default is 1.
    intensity_from
        Set lower threshold for peak intensity. Default is 10.0.
    loss_mz_from
        Minimum allowed m/z value for losses. Default is 0.0.
    loss_mz_to
        Maximum allowed m/z value for losses. Default is 1000.0.
```

Reply:
It is usually good to add what a user could enter as input parameter, so I added the list. If you agree you can simply click ``commit suggestion``. You can also edit it before.